contract_call_query_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 )
 
-func TestNewContractCallQuery_Execute(t *testing.T) {
+func TestIntegrationNewContractCallQuery(t *testing.T) {
```

Comment:
incorrect test name, name of the struct isn't `NewContractCallQuery`, it's just `ContractCallQuery`, so the name should be `TestIntegrationContractCallQuery*`